MMM-HideShowMove.js

```diff
@@ -207,6 +209,12 @@ Module.register("MMM-HideShowMove", {
       case moveTopic:
         this.moveModule(payload);
         break;
+      case 'DOM_OBJECTS_CREATED':
+        if(this.config.Startpage != ""){
+            MM.getModules().exceptWithClass(this.config[this.config.Startpage]).exceptWithClass(this.name).enumerate(module => module.hide());
```

Comment:
I really like this idea and did not know these helpers exists. Is there any advantage in excluding this.name? It has nothing to show, so I suppose it should not really make a difference to hide/show it?

Reply:
I tested it without this.name but i got every time the output that the module MMM-HideShowMove is suspended. So i thought it would be better to exclude it. I think in general it makes no difference.